e2e/cypress/integration/account_settings/display/account_settings_positiion_spec.js

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @account_setting
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Account Settings > General > Position', () => {
+    let testTeam;
+    let testUser;
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+        });
+    });
+
+    it('MM-T2063 Position / Can enter position', () => {
+        cy.apiLogin(testUser);
+
+        // # Visit town square
+        cy.visit(`/${testTeam.name}/channels/town-square`);
```

Comment:
nit: Can be moved to `beforeEach` if all `it` blocks have the same starting point.